upup/pkg/fi/cloudup/azuretasks/vmscaleset.go

```diff
@@ -60,6 +60,35 @@ func ParseSubnetID(s string) (*SubnetID, error) {
 	}, nil
 }
 
+// LoadBalancerID contains the resource ID/names required to construct a loadbalancer ID.
+type LoadBalancerID struct {
```

Comment:
We should probably not export this to make it clear that its not relevant outside of this package.

```suggestion
// loadBalancerID contains the resource ID/names required to construct a loadbalancer ID.
type loadBalancerID struct {
```

Reply:
made the change. 

no worries! i appreciate all the help and feedback you've provided me with during the code reviewing process.